apistructs/pipeline_label.go

```diff
@@ -52,3 +56,35 @@ type TargetIDSelectByLabelRequest struct {
 	// OrderByTargetIDAsc 根据 target_id 升序，默认为 false，即降序
 	OrderByTargetIDAsc bool `json:"orderByTargetIDAsc"`
 }
+
+type PipelineLabelBatchInsertResponse struct{}
+
+type PipelineLabelBatchInsertRequest struct {
+	Labels []PipelineLabel `json:"labels"`
+}
+
+type PipelineLabelListRequest struct {
+	PipelineSource  PipelineSource `json:"pipelineSource"`
+	PipelineYmlName string         `json:"pipelineYmlName"`
+	TargetIDS       []uint64       `json:"targetIds"`
+	Keys            []string       `json:"keys"`
+
+	FilterTargetIDNotEqualKeys []string `json:"filterTargetIDNotEqualKeys"`
```

Comment:
split into two request.
one request with `Keys` field for dbclient.
another request with `FilterTargetIDNotEqualKeys` for api use.

Reply:
MatchKeys
UnMatchKeys